Recrutify/Recrutify.Services/Events/UpdateStatusEventArgs.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using Recrutify.Services.DTOs;
+
+namespace Recrutify.Services.Events
+{
+    public class UpdateStatusEventArgs : EventArgs
+    {
+        public IEnumerable<Guid> Ids { get; set; }
```

Comment:
rename how I suggested CandidatesIds, CandidateStatus,
it's not clear whose ids, whose status